frontend/portals/PdpRechargeProducts/index.jsx

```diff
@@ -48,7 +49,7 @@ const PdpRechargeProducts = ({
       <RechargeOption
         key={id}
         id={id}
-        chargeIntervalFrequency={charge_interval_frequency}
+        chargeIntFreq={charge_interval_frequency}
         cutoffDayOfMonth={cutoff_day_of_month}
         cutoffDayOfWeek={cutoff_day_of_week}
         discountAmount={discount_amount}
```

Comment:
check out the decomposition of subscription_defaults starting on line 30. The expire_after_spefific_number_of_charges property has a spelling error in it. Maybe we don't use this value anyway so it can simply be removed.

Reply:
fixed